lib/server.js

```diff
@@ -1,13 +1,18 @@
 const express = require('express')
+const responseTime = require('response-time')
 const path = require('path')
 
 module.exports = function (webhook) {
   const app = express()
+  app.set('x-powered-by', false)
```

Comment:
I encourage you to read [this statement](https://github.com/expressjs/express/pull/2813#issuecomment-159270428) from an Express maintainer about this issue.

Reply:
I don't disagree with his point about security by obscurity.

I would be more sympathetic to the "give credit to the hard-working people who have enabled you to write your code so fast" argument if it also acknowledged all the tools that express has used for free that enabled it to receive such prominence (like Node.js, npm and GitHub for hosting the project for free, Sinatra and rack from ruby since that heavily influence the design, etc).

The header is still frivolous, and there are better ways to acknowledge people who made it possible.

Reply:
I'm all for surfacing the packages we use somewhere to give ample credit (they certainly deserve it), but I don't think anyone looks at headers and says "Hey what's Express?" - so I'm okay with doing this.